pkg/webhook/advancedcronjob/validating/advancedcronjob_create_update_handler_test.go

```diff
@@ -407,6 +407,296 @@ func TestAdvancedCronJobCreateUpdateHandler_Handle(t *testing.T) {
 			expectedResult: false,
 			expectedError:  true,
 		},
+		{
+			name: "create v1beta1 AdvancedCronJob ImageListPullJobTemplate",
```

Comment:
please add ut to allow changes of imagelistpulljob template or other immutable fields